src/setops.jl

```diff
@@ -28,3 +28,77 @@ function _decompose_1D(X0::LazySet{N}) where {N}
     end
     return CartesianProductArray(out)
 end
+
+# ==============================================================
+# Construct the initial states for the continuous post-operator
+# ==============================================================
+
+using ReachabilityAnalysis: TaylorModel1, TaylorModelN, fp_rpa, zeroBox, symBox
+
+abstract type AbstractReconstructionMethod end
+
+struct ZonotopeReconstructor <: AbstractReconstructionMethod end
+
+function _reconstruct(method::ZonotopeReconstructor, P₀::LazySet, U₀::LazySet, X, ti)
+    Q₀ = P₀ × U₀
```

Comment:
I think there is nothing specific to zonotopes. Maybe call it `CartesianProductReconstructor`?